flask/.gitignore

```diff
@@ -1,3 +1,4 @@
 __pycache__/
 venv/
 *.pyc
+.devcontainer.json
```

Comment:
Is this from the VSCode extension?

Reply:
Yes -- it's the settings for vscode while it's attached to a docker container (extensions to load, paths, etc.).

Reply:
I think you need to activate the EditorConfig for VSCode extension as this file is missing a trailing newline but the editorconfig takes care of that.